dataprofiler/tests/profilers/utils.py

```diff
@@ -13,6 +15,7 @@ def set_seed(seed=None):
     """
     np.random.seed(seed)
     random.seed(seed)
+    dp.set_seed(seed)
```

Comment:
Why is this written for profilers and labelers when its the same function? Is there a reason this isn't more centrally located?